Assets/Resources/AI_Server.txt

```diff
@@ -1,3 +0,0 @@
-AI_API_URL=http://91.107.198.4:8001/
```

Comment:
you deleted deleted files with private data, but did not delete `SketchfabSettings.asset`